### PR TITLE
Add a working follow button to business pages 

### PR DESCRIPTION
**Pull request by lukasberglund**

### CODE REVIEW

File: capstone/src/main/webapp/js/util.js [hunk 1]
```diff
@@ -108,11 +108,21 @@ export function getJsonObject(url, parameters = {}) {
...
+  
+  return fetch(url, {method: type});
+}
+
 export function makePostRequest(url, parameters) {
-  return fetch('https://example.com/profile', {
-    method: 'POST',
-    body: JSON.stringify(parameters),
-  });
+  return makeRequest(url, parameters, 'POST');
```
Comment (alygerharter) [inline capstone/src/main/webapp/js/util.js:125]: no need to add an extra nested function. just put the contents of makeRequest here.

Comment (lukasberglund) [reply capstone/src/main/webapp/js/util.js:125]: Why is that better form?

Comment (alygerharter) [reply capstone/src/main/webapp/js/util.js:125]: you aren't reusing makeRequest() anywhere, so then you just have a function around a function.  it's not really adding anything, here.

Comment (lukasberglund) [reply capstone/src/main/webapp/js/util.js:125]: I do use it in `business.js` on line 203. If you want I can get rid of makePostRequest and replace those uses with `makeRequest(url, parameters, 'POST');` though.

Comment (lukasberglund) [reply capstone/src/main/webapp/js/util.js]: Update: I removed `makePostRequest()`

Comment (alygerharter) [reply capstone/src/main/webapp/js/util.js]: this is cleaner, thanks!

**Pull request merged**